portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -38,6 +38,90 @@ definition {
 		}
 	}
 
+	@description = "LPS-122873 When a new category is created and assigned to a web content, the relative ordering of categories remain the same and the new category gets a new color, positionated in the last position"
+	@priority = "3"
+	test CheckTheOrderWithANewCategory {
+		property portal.acceptance = "true";
+
+		task ("Create a list of Audiences") {
+			for (var count : list "1,2,3") {
+				JSONCategory.addCategory(
+					groupName = "Global",
+					title = "S ${count}",
+					vocabularyName = "Audience");
+			}
+		}
+
+		task ("Create a list of Stages") {
+			for (var count : list "1,2,3") {
+				JSONCategory.addCategory(
+					groupName = "Global",
+					title = "test ${count}",
+					vocabularyName = "Stage");
+			}
+		}
+
+		task ("Create a list of Web Content") {
+			for (var count : list "1,2,3") {
+				JSONWebcontent.addWebContent(
+					content = "Webcontent Content",
+					groupName = "Test Site Name",
+					title = "Webcontent Title ${count}");
+			}
+		}
+
+		task ("Select of categories") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+			for (var count : list "1,2,3") {
+				WebContentNavigator.gotoEditCP(webContentTitle = "Webcontent Title ${count}");
+
+				for (var count : list "1,2,3") {
+					WebContent.selectCategory(
+						categoryName = "S ${count}",
+						vocabularyName = "Audience");
+				}
+
+				for (var count : list "1,2,3") {
+					WebContent.selectCategory(
+						categoryName = "test ${count}",
+						vocabularyName = "Stage");
+				}
+
+				PortletEntry.publish();
+			}
+		}
+		task ("Navigate to content Dashboard and View the order of categories") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Content",
+				panel = "Applications",
+				portlet = "Content Dashboard");
+		}
+
+		task ("Create a new category and assign one of the contents") {
+			JSONCategory.addCategory(
+				groupName = "Global",
+				title = "test 4",
+				vocabularyName = "Stage");
+
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+			WebContentNavigator.gotoEditCP(webContentTitle = "Webcontent Title 1");
+
+			WebContent.selectCategory(
+				categoryName = "test 4",
+				vocabularyName = "Stage");
+
+			PortletEntry.publish();
+		}
+		task ("Navigate to the content panel and see if, in the new category order, the new category receives a new color, placed in the last position") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Content",
+				panel = "Applications",
+				portlet = "Content Dashboard");
+		}
```

Comment:
Here you say that it navigates and check the category order, color and last position, but you are just navigating. Where are the asserts?